apps/omg/lib/omg/utxo/exit.ex

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Utxo.Exit do
```

Comment:
I have two comments here and I'm a little unclear on the purpose of the module
1/ name/moduledoc: either we mean a `Utxo.StandardExit` and want this to only ever represent SEs **or** we mean a generic representation of a UTXO-exiting event, i.e. we want other flavors of exits use this too. In the latter case I'd keep the generic `Exit` name and explain purpose in the `moduledoc`
2/ location: slightly dependant to (1/) - an alternative location would be somewhere around `OMG.Watcher.DB`, as it seems to be specific to that code. Encouraging e.g. `omg_child_chain` to leverage this in the future could lead to unwanted coupling.

Reply:
1/ To be honest, I named it that way, because my head was deep in the code trying to get `OMG.Watcher.DB.EthEvent.insert_exit` to work.

But now when I take a stop back, it doesn't need to be called `Exit` at all. It's really just representing an EthEvent/UTXO struct. It could be used for any type of event.  I could just name it `EthEventUTXO`. Not exactly beautiful :smile:, but it is the correct meaning. Any other suggested names? I'm open...

2/ Where do you recommend the best place to put it is? I put it here because this is also where `OMG.Utxo` and `OMG.Utxo.Position` were already defined. I'd be fine to move it into 
`OMG.Watcher.DB`.

Reply:
Oooh, I see you're not even really using this struct (except type specs, which are conflicting with the code - take a look in `ethevent.ex`, you're really using a map with different field names :) ). 

If it is a helper named pair structure that is only used to move data around that `DB.EthEvent` module, I think it'd be fine to just use a tuple, and scrap this module altogether.

Reply:
Removed!